tests/test_process_definition.py

```diff
@@ -0,0 +1,145 @@
+from os import path
+import json
+
+import pytest
+from referencing import Resource
+
+from cirro.models import process
+
+data_path = path.join(path.dirname(__file__), 'data')
+
+def test_pipeline_definition_nextflow_simple():
```

Comment:
can you use the `unittest.TestCase` pattern like the other tests (and `self.assertEquals` etc.)